### ISSUE TITLE
Dynamic Min/Max TemplateOptions

### ISSUE THREAD

**Issue by holtjonathan** [formly-js/angular-formly#491]
I created the following SO question: http://stackoverflow.com/questions/32827607/updating-the-min-max-templateoptions

@ckniffen suggested that this is a future enhancement as the template manipulator cannot set dynamic values to min/max.

**Comment by kentcdodds** [comment]
Answered [here](http://stackoverflow.com/a/32829953/971592). The GitHub issues in this project are reserved for reporting bugs and feature requests. In the future, please follow the instructions here to get help: http://help.angular-formly.com

**Comment by holtjonathan** [comment]
My bad man.  I literally thought that this was an issue.  Thanks for the example!

**Comment by kentcdodds** [comment]
You're good. You thought it was a feature request. No harm done :-) Good luck!

**Comment by holtjonathan** [comment]
FYI...you are a wizard.  Just putting that out there.

**Comment by kentcdodds** [comment]
Awe, you're making me :blush: Thanks!

**Comment by holtjonathan** [comment]
One last question...I modifed your example to pass hard coded data: http://jsbin.com/juqihizefo/edit?js,output

This works totally fine on the jsbin but locally, only your way works (checking the formState.changed).  If I set `'templateOptions.min': 'formState.min'` then to.min is ALWAYS blank.

I feel like I am missing something obvious.  I can't recreate this issue on jsbin because it works there but not local.  :(
